test/onnx/gen_onnx.py

```diff
@@ -5017,6 +5017,31 @@ def prelu_brcst_test():
     return ([node], [arg0, arg1], [arg_out])
 
 
+@onnx_test()
+def qlinearconv_test():
```

Comment:
Need to add test for parser inside onnx_test.cpp

Reply:
Done.